[TW-1174] Doc refresh for "Discussing your work" (#5160)
postmanlabs/postman-docs: src/pages/docs/collaborating-in-postman/working-with-your-team/discussing-your-work.md

## Contents

* [Comment on a collection, folder, or request](#comment-on-a-collection-folder-or-request)
* [Reply to comments](#reply-to-comments)
* [Resolve comments](#resolve-comments)
* [Mention teammates in comments](#mention-teammates-in-comments)

## Comment on a collection, folder, or request

You can add comments to a collection in Postman. You can also add comments to requests or folders in collections. When you add a comment, Postman notifies other collection contributors about your comment.

> You must have access to a collection to add comments to it.

<!-- -->

> > Postman supports Markdown in comments. See the [Markdown Cheatsheet on GitHub](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for tips on formatting.

Reviewer: To avoid the double note, could consider moving this one under the second list item (indented twice)
<img width="642" alt="Screenshot 2023-10-19 at 1 17 46 PM" src="https://github.com/postmanlabs/postman-docs/assets/21180383/5b251b12-31ba-462f-9e90-700bc9b9541b">

Author: Good suggestion, I'll go with that.

Author: See b0d471154b7ecdef6550e31d4313cc8e9a643363

To add a comment to a collection, folder, or request, do the following:

1. Select **Collections** in the sidebar and select the collection, folder, or request you want to comment on.
1. Select the comments icon <img alt="Comments icon" src="https://assets.postman.com/postman-docs/icon-comments-v9.jpg#icon" width="18px"> in the right sidebar and enter your comment.
1. (Optional) Select the **Watch collection** checkbox to be notified when there are changes to the collection.
1. Select **Comment** to add your comment.

> ![Commenting on a collection](https://assets.postman.com/postman-docs/v10/commenting-on-a-collection-v10-18b.jpg)

Author: @claireriley Is this what you were thinking?
![commenting-on-a-collection-v10-18c](https://github.com/postmanlabs/postman-docs/assets/113702138/58143037-b88c-41e8-9ef6-ffc30104f075)

> ![Commenting on a collection](https://assets.postman.com/postman-docs/v10/commenting-on-a-collection-v10-18b.jpg)

Reviewer: @akinard-postman that works or could even do just this, whichever you think looks best in context!
<img width="573" alt="Screenshot 2023-10-19 at 6 22 42 PM" src="https://github.com/postmanlabs/postman-docs/assets/21180383/fcb24c15-7949-4fd2-9b18-bbc7a426f5f6">

> ![Commenting on a collection](https://assets.postman.com/postman-docs/v10/commenting-on-a-collection-v10-18b.jpg)

Author: I'll go with the one you suggested. I checked both and the one you suggest looks better. Thanks!

1. (Optional) Select the **Watch collection** checkbox to be notified when there are changes to the collection.
1. Select **Comment** to add your comment.

    ![Adding an inline comment](https://assets.postman.com/postman-docs/v10/commenting-inline-v10-16a.jpg)

### Edit, delete, or link to a comment

You can edit or delete a comment, or a get a link to a comment you can share with others.

1. Select a collection, folder, or request in the sidebar, then select the comments icon <img alt="Comments icon" src="https://assets.postman.com/postman-docs/icon-comments-v9.jpg#icon" width="18px"> in the right sidebar.
1. Select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to a comment and do one of the following:

    * Select **Copy link** to copy a direct link to the comment to your clipboard.
    * Select **Edit** to make changes to the comment.
    * Select **Delete** to delete the comment.

<img alt="Managing a comment" src="https://assets.postman.com/postman-docs/v10/commenting-managing-v10-16.jpg" width="443px" />

> > If you have the [Admin role](/docs/collaborating-in-postman/roles-and-permissions/#team-roles) in the workspace, you can delete contributor comments, but you can't edit contributor comments.

Reviewer: ```suggestion
> If you're a [Workspace Admin](/docs/collaborating-in-postman/roles-and-permissions/#team-roles), you can delete contributor comments, but you can't edit contributor comments.
```

## Contents

* [Commenting on a collection, folder, or request](#commenting-on-a-collection-folder-or-request)
* [Replying to comments](#replying-to-comments)
* [Resolving comments](#resolving-comments)
* [Mentioning teammates in comments](#mentioning-teammates-in-comments)

> ## Commenting on a collection, folder, or request

Reviewer 2: You could consider changing all headings from gerunds to imperatives per our new [heading guidelines](https://postmanlabs.atlassian.net/wiki/spaces/TW/pages/4576804916/Headings+in+the+Learning+Center)

Author: Updated the headings in deaaae2747812acc7254b4f9e905363b0aa6503a -- thanks for catching that!

You can add comments to a collection in Postman. You can also add comments to requests or folders in collections. When you add a comment, Postman notifies other collection contributors about your comment.

> You must have access to a collection to add comments to it.

<!-- -->

> Postman supports Markdown in comments. See the [Markdown Cheatsheet on GitHub](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for tips on formatting.

To add a comment to a collection, folder, or request, do the following:

1. Select **Collections** in the sidebar and select the collection, folder, or request you want to comment on.
1. Select the comments icon <img alt="Comments icon" src="https://assets.postman.com/postman-docs/icon-comments-v9.jpg#icon" width="18px"> in the right sidebar and enter your comment.
1. (Optional) Select the **Watch collection** checkbox to be notified when there are changes to the collection.
1. Select **Comment** to add your comment.

> ![Commenting on a collection](https://assets.postman.com/postman-docs/v10/commenting-on-a-collection-v10-18a.jpg)

Reviewer 2: Does this screenshot need a border? If it has one, it's very thin. One thing that might help is making the Postman window a little narrower, so everything's a bit bigger.

Also, I don't see the "Watch collection" checkbox in the screenshot. I remember this was sometimes tricky to get the checkbox to show up. I feel like it only was there consistently for the first comment in a new collection.

Author: Good call; I'll redo the screenshot for this one.

### Adding inline comments to a request

> Use inline comments to comment on a specific part of a request. You can add comments to parameters, headers, and request bodies. You can also add comments to parts of a pre-request script or a test script. When there are new inline comments, an notification appears on the comments icon <img alt="Inline comments icon" src="https://assets.postman.com/postman-docs/v10/icon-inline-comment-v10-16.jpg#icon" width="22px"> in the request pane.

Reviewer 2: ```suggestion
Use inline comments to comment on a specific part of a request. You can add comments to parameters, headers, and request bodies. You can also add comments to parts of a pre-request script or a test script. When there are new inline comments, a notification appears on the comments icon <img alt="Inline comments icon" src="https://assets.postman.com/postman-docs/v10/icon-inline-comment-v10-16.jpg#icon" width="22px"> in the request pane.
```
